preprocess.py

```diff
@@ -119,6 +131,7 @@
 }
 logging.info(f"Added schema for 200 response of /status/ get")
 
+# NOTE: below fixed upstream
```

Comment:
Does this mean that that part of the preprocessing code is no longer required? If so, it should be removed. I will merge regardless, to get https://github.com/e-breuninger/terraform-provider-netbox/pull/358 ahead.

Reply:
Yes, see swagger.json, line 79870 - I'm pretty sure that's now exactly the same as the fix, so the fix can be removed